src/docs/sdk/event-payloads/properties/status.mdx

```diff
@@ -0,0 +1,30 @@
+`status`:
+
+: _Optional_. Describing the `status` of the Span. We use this property to determine if the [Transaction](./transaction)
+ended in an error or not.
+
+State|Description|HTTP status code equivalent
+---|---|---
+`ok`|Not an error, returned on success|200 and 2XX HTTP statuses
+`cancelled`|The operation was cancelled, typically by the caller|499
+`unknown`|An unknown error raised by APIs that don't return enough error information|500
```

Comment:
This can also be `unknown_error` -> https://github.com/getsentry/relay/blob/master/relay-general/src/protocol/contexts.rs#L503

Reply:
probably describe it as "unknown result" vs "unknown error"